AndelaOSP/wirebot
# #153113987 Post Incident to API

#### What does this PR do?
This PR posts an incident to the API when it's submitted via the `Slack bot`

#### Description of Task to be completed?
- When an incident is reported using the `Slack bot`, the incident's details are posted to the API and consequently created on the backend

#### How should this be manually tested?
Given that you are signed in to a Slack workspace where the bot has been installed:
- Initiate a conversation with the bot
- Indicate that you want to report an incident
- Proceed through the steps of reporting an incident
- Confirm and submit the incident
- Confirm on the `backend` that the incident has been created

#### Any background context you want to add?
This will allow for the creation of reported incidents on the backend, whose data can then be used for other applications such as the frontend

#### Screenshots
N/A

#### What are the relevant pivotal tracker stories?
[#153113987](https://www.pivotaltracker.com/n/projects/2117172/stories/153113987)
